oximeter/oximeter/schema/instance-data-link.toml

```diff
@@ -0,0 +1,86 @@
+format_version = 1
+
+[target]
+name = "instance_data_link"
+description = "A network data link related to a guest/instance"
+authz_scope = "fleet"
+versions = [
+    { version = 1, fields = [ "interface_id", "instance_id", "project_id", "silo_id", "sled_id", "sled_model", "sled_revision", "sled_serial" ] },
+]
+
+[[metrics]]
+name = "bytes_sent"
+description = "Total number of bytes sent on the data link"
+units = "bytes"
+datum_type = "cumulative_u64"
+versions = [
+    { added_in = 1, fields = [] }
+]
+
+[[metrics]]
+name = "bytes_received"
+description = "Total number of bytes received on the data link"
+units = "bytes"
+datum_type = "cumulative_u64"
+versions = [
+    { added_in = 1, fields = [] }
+]
+
+[[metrics]]
+name = "packets_sent"
+description = "Total number of packets sent on the data link"
+units = "count"
+datum_type = "cumulative_u64"
+versions = [
+    { added_in = 1, fields = [] }
+]
+
+[[metrics]]
+name = "packets_received"
+description = "Total number of packets received on the data link"
+units = "count"
+datum_type = "cumulative_u64"
+versions = [
+    { added_in = 1, fields = [] }
+]
+
+[[metrics]]
+name = "packets_dropped"
+description = "Number of packets dropped on the RX queue of the data link"
+units = "count"
+datum_type = "cumulative_u64"
+versions = [
+    { added_in = 1, fields = [] }
+]
+
+[fields.interface_id]
+type = "uuid"
+description = "The ID of the link associated with the virtual network interface it is attached to"
+
+[fields.instance_id]
+type = "uuid"
+description = "The ID of the virtual machine instance this link is attached to"
+
+[fields.project_id]
+type = "uuid"
+description = "The ID of the virtual machine instance's project this link is attached to"
+
+[fields.silo_id]
+type = "uuid"
+description = "The ID of the virtual machine instance's silo this link is attached to"
```

Comment:
These are also kind of awkward since "ID" and the thing the ID refers to are really far apart. Perhaps: "The ID of the {silo,project} containing the virtual machine instance this link is attached to"?